odoo-shippable/scripts/build-image.sh

```diff
@@ -161,7 +172,14 @@ EOF
 
 cat >> ~/.vimrc.before << EOF
 let g:spf13_bundle_groups = ['general', 'writing', 'odoovim',
-                           \ 'programming', 'php', 'ruby',
+                           \ 'programming', 'youcompleteme', 'php', 'ruby',
+                           \ 'python', 'javascript', 'html',
+                           \ 'misc']
+EOF
+
+cat >> ~/.vimrc.before.local << EOF
```

Comment:
@rdmachadog 
Should we use `~/.vimrc.before.local` and `~/.vimrc.before` or just use one file?
If is just one what is better for you?